### ISSUE TITLE
Type mismatch causing build errors

### ISSUE THREAD

**Issue by vendion** [d2r2/go-shell#3]
I'm trying to use your 'go-i2c' and `go-sht3x` packages for a project which one or both of those packages indirectly depends on this according to `go mod` but there seems to be a type mismatch that causes this package to fail to build.

Output when fetching packages:
```
go: finding github.com/d2r2/go-sht3x latest
go: finding github.com/d2r2/go-logger latest
go: finding github.com/d2r2/go-shell latest
go: finding github.com/d2r2/go-i2c latest
go: downloading github.com/d2r2/go-i2c v0.0.0-20181113114621-14f8dd4e89ce
go: downloading github.com/d2r2/go-logger v0.0.0-20181221090742-9998a510495e
go: downloading github.com/d2r2/go-shell v0.0.0-20190508080434-6fd313082bbf
go: extracting github.com/d2r2/go-i2c v0.0.0-20181113114621-14f8dd4e89ce
go: extracting github.com/d2r2/go-shell v0.0.0-20190508080434-6fd313082bbf
go: extracting github.com/d2r2/go-logger v0.0.0-20181221090742-9998a510495e
# github.com/d2r2/go-shell
/home/vendion/gocode/pkg/mod/github.com/d2r2/go-shell@v0.0.0-20190508080434-6fd313082bbf/utils.go:40:23: invalid operation: stat.Bavail * uint64(stat.Bsize) (mismatched types int64 and uint64)
```

Go and OS info:
```
(go version output)
go version go1.13.4 freebsd/amd64

(go env output)
GO111MODULE=""
GOARCH="amd64"
GOBIN=""
GOCACHE="/home/vendion/.cache/go-build"
GOENV="/home/vendion/.config/go/env"
GOEXE=""
GOFLAGS=""
GOHOSTARCH="amd64"
GOHOSTOS="freebsd"
GONOPROXY=""
GONOSUMDB=""
GOOS="freebsd"
GOPATH="/home/vendion/gocode"
GOPRIVATE=""
GOPROXY="https://proxy.golang.org,direct"
GOROOT="/usr/local/go"
GOSUMDB="sum.golang.org"
GOTMPDIR=""
GOTOOLDIR="/usr/local/go/pkg/tool/freebsd_amd64"
GCCGO="gccgo"
AR="ar"
CC="clang"
CXX="clang++"
CGO_ENABLED="1"
GOMOD="/usr/home/vendion/Projects/i2c-test/go.mod"
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -m64 -pthread -fno-caret-diagnostics -Qunused-arguments -fmessage-length=0 -fdebug-prefix-map=/tmp/go-build283462116=/tmp/go-build -gno-record-gcc-switches"
```

**Comment by d2r2** [comment]
The source of issue in how Statfs_t structure declared in Linux:
```

type Statfs_t struct {
	Type    int64
	Bsize   int64
	Blocks  uint64
	Bfree   uint64
	Bavail  uint64
	Files   uint64
	Ffree   uint64
	Fsid    Fsid
	Namelen int64
	Frsize  int64
	Flags   int64
	Spare   [4]int64
}
```
, and FreeBSD:
```
type Statfs_t struct {
	Version     uint32
	Type        uint32
	Flags       uint64
	Bsize       uint64
	Iosize      uint64
	Blocks      uint64
	Bfree       uint64
	Bavail      int64
	Files       uint64
	Ffree       int64
	Syncwrites  uint64
	Asyncwrites uint64
	Syncreads   uint64
	Asyncreads  uint64
	Spare       [10]uint64
	Namemax     uint32
	Owner       uint32
	Fsid        Fsid
	Charspare   [80]int8
	Fstypename  [16]int8
	Mntfromname [1024]int8
	Mntonname   [1024]int8
}
```

**Comment by d2r2** [comment]
Issue resolved in #4. Now it should works both well on Linux and FreeBSD.
